editor/export/editor_export.cpp

```diff
@@ -83,16 +85,21 @@ void EditorExport::_save() {
 		config->set_value(section, "encryption_exclude_filters", preset->get_enc_ex_filter());
 		config->set_value(section, "encrypt_pck", preset->get_enc_pck());
 		config->set_value(section, "encrypt_directory", preset->get_enc_directory());
-		config->set_value(section, "script_encryption_key", preset->get_script_encryption_key());
+		credentials->set_value(section, "script_encryption_key", preset->get_script_encryption_key());
 
 		String option_section = "preset." + itos(i) + ".options";
 
 		for (const PropertyInfo &E : preset->get_properties()) {
-			config->set_value(option_section, E.name, preset->get(E.name));
+			if ((E.usage | PROPERTY_USAGE_SECRET) == E.usage) {
+				credentials->set_value(option_section, E.name, preset->get(E.name));
+			} else {
+				config->set_value(option_section, E.name, preset->get(E.name));
+			}
 		}
 	}
 
 	config->save("res://export_presets.cfg");
+	credentials->save("res://export_credentials.cfg");
```

Comment:
Can we put this inside of the `.godot/` folder? We want to heavily encourage this being not committed to version control. Considering that, we also want to encourage it being stored elsewhere, so placing it inside the folder meant to be deleted without consequences encourages storing export credentials elsewhere.

Reply:
Do we want to encourage people to not commit it or do we want to force them? Storing it in the `.godot` folder would basically do the latter. I like having the files responsible for exporting in proximity to each other and adding `export_credentials.cfg` to the default `.gitignore` file might be enough.

Reply:
Yes, I think it would be better to (soft-)force the file to not be committed. Also, I would prefer the `.gitignore` file to only need to contain one entry, the `.godot/` folder.

Reply:
Done